co-operative.md

```diff
@@ -0,0 +1,50 @@
+# Hypha Worker Co-operative
+
+## About Us 
+
+Hypha Worker Co-operative Inc. is a worker co-operative incorporated on August 
+1, 2019 in Ontario, Canada (Ontario Corporation No. 5019866).
+
+Hypha is incorporated as a **non-share capital** worker co-operative under the 
+[*Ontario Co-operative Corporations Act, 1990*][coop-act]. This entails specific 
+conditions on our primary object, size, and role of members, as addressed in 
+Article [1], [1.1], [144] and [145.1] of the Act.
+
+We are a non-profit organization. 
```

Comment:
```suggestion
We are a non-profit _organization_. As per 2017's [_Cutting Unnecessary Red Tape Act_](https://www.ontario.ca/laws/statute/s17020), Hypha falls under neither the current [_Corporations Act_](https://www.ontario.ca/laws/statute/90c38), nor the upcoming [_Not-for-Profit Corporations Act_](https://www.ontario.ca/laws/statute/10n15), which govern not-for-profit _corporations_ in Ontario.
```

Reply:
I found a resource that explains the proposed changes in the new law and what isn't impacted. I should dig that up for here

Reply:
I am still confused after reading @patcon's suggestion because it says what we _don't_ fall under and not what we _do_ fall under 😆

Reply:
heh :) I think that's maybe an artefact of the suggestion mode on PRs, no? This is in the paragraph after we say what we fall under

![Screen Shot 2019-11-28 at 1 10 09 PM](https://user-images.githubusercontent.com/305339/69826276-7ca8b880-11e0-11ea-81d9-952ab6dcb285.png)

Only suggesting because this distinction feels like hard-earned knowledge that took us a long time to sort out! Would love to capture somewhere, but maybe this is wrong place. Any suggestions?